src/chat/dto/update-chat.dto.ts

```diff
@@ -2,11 +2,11 @@ import {
   IsArray,
   IsBoolean,
   IsDate,
-  IsNumber,
   IsOptional,
+  IsString,
   ValidateNested,
 } from 'class-validator';
-import { Transcript } from '../transcript/entity/transcript.entity';
+import { Message } from '../../../libs/common/src/entities/message.entity';

```

Comment:
Why is this in common folder?

Reply:
I am using the guards in all of the app

Reply:
You've put Message in common folder. Shouldn't be there.